scripts/20170523-col913/create_table_pinned_user_assets.sql

```diff
@@ -0,0 +1,17 @@
+-- Linking table records assets pinned by users
+
+CREATE TABLE pinned_user_assets (
+  asset_id integer NOT NULL REFERENCES assets (id),
+  user_id integer NOT NULL REFERENCES users (id),
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL
```

Comment:
@johncrossman It looks like our other many-to-many join tables (`asset_users`, `asset_categories`, `whiteboard_elements`) also include an `updated_at` timestamp. I assume that's a Sequelize default; any reason not to include it here?